public/scripts/coffeePageView.js

```diff
@@ -0,0 +1,71 @@
+(function(module) {
+
+  var coffeeObj = [];
+
+  function Coffee(infoObj) {
+    this.contryName = infoObj.contryName;
+    this.countryBio = infoObj.countryBio;
+    this.featuredCoffeePic = infoObj.featuredCoffeePic;
+    this.featuredRoasterBio = infoObj.featuredRoasterBio;
+    this.map = infoObj.map;
+    this.brewPic = infoObj.brewPic;
+    this.brewInstructions = infoObj.brewInstructions;
+  };
+  Coffee.prototype.toHtml = function() {
+    var templateSrc = $("#coffeePageTemplate").text();
+    var template = Handlebars.compile(templateSrc);
+    return template(this);
+  };
+
+  Coffee.all = [];
+
+  Coffee.loadAll = function(rawData) {
```

Comment:
This is where you could take those static files for each country and make a call to a separate table/model in your DB.